README.rst

```diff
@@ -39,8 +39,7 @@ Then get the frontend building::
    $ yarn install
    $ yarn serve
 
-Then, you can visit the local site at ``http://localhost:8000/``. The
-terminal output will say ``:8080``, but ignore it, it's a liar.
+Then, you can visit the local site at ``http://localhost:8000/``.
```

Comment:
Please revert this change, and the change to `package.json`.

Here's the reason: the app is mostly server-side rendered, with the JS components served by the backend Django process and the `js:serve` script not proxying back to the Django process. The Django process runs on port `8000` (by default). `yarn serve` will run both `js:serve` and `py:serve` in parallel, and the Python process will claim port 8000. The JS process will then increment until it finds a free port, probably landing on port 8001. Therefore, it is (a) pointless to set it to use port `8000`, (b) still true that the terminal output is a liar (just lying by saying `8001` instead of `8080`).

Reply:
I did test this change before committing, and it does work over here; the terminal says port 8000, and my browser successfully connects to the site on port 8000; the JS process appears to realize it's not really using a port and thus doesn't bother checking and incrementing. I'll certainly revert that last commit, or you can, if you do think it might cause port collisions in the future, though.

Reply:
Ok, well, it did just display port 8001 once, although the vast majority of the time it accurately displays port 8000; apparently it's capable of switching back and forth even within the same terminal session:
![image](https://user-images.githubusercontent.com/49729978/74599261-01bba500-504d-11ea-9248-63f77c937e6d.png)
So that's annoying, and I'll be reverting this right away, sorry for doubting you

Reply:
Haha, it's OK! It's just a thing that I encountered myself and didn't want to put the effort into resolving, so I just left that comment in the Readme.